quartz/components/Body.tsx

```diff
@@ -1,13 +1,50 @@
 // @ts-ignore
 import clipboardScript from "./scripts/clipboard.inline"
 import clipboardStyle from "./styles/clipboard.scss"
+// @ts-ignore
+import mermaidScript from "./scripts/mermaid.inline"
+import mermaidStyle from "./styles/mermaid.scss"
 import { QuartzComponent, QuartzComponentConstructor, QuartzComponentProps } from "./types"
 
 const Body: QuartzComponent = ({ children }: QuartzComponentProps) => {
-  return <div id="quartz-body">{children}</div>
+  return (
+    <div id="quartz-body">
+      {children}
+      <div id="mermaid-container">
+        <div class="mermaid-backdrop" />
+        <div id="mermaid-space">
+          <div class="mermaid-header">
+            <button
+              class="close-button"
+              aria-label="close button"
+              title="close button"
+              type="button"
+            >
+              <svg
+                aria-hidden="true"
+                xmlns="http://www.w3.org/2000/svg"
+                width="24"
+                height="24"
+                viewBox="0 0 24 24"
+                fill="none"
+                stroke="currentColor"
+                strokeWidth="2"
+                strokeLinecap="round"
+                strokeLinejoin="round"
+              >
+                <line x1={18} y1={6} x2={6} y2={18} />
+                <line x1={6} y1={6} x2={18} y2={18} />
+              </svg>
+            </button>
+          </div>
+          <div class="mermaid-content" />
+        </div>
+      </div>
+    </div>
+  )
 }
 
-Body.afterDOMLoaded = clipboardScript
-Body.css = clipboardStyle
+Body.afterDOMLoaded = clipboardScript + mermaidScript
+Body.css = clipboardStyle + mermaidStyle
```

Comment:
we should do this in components/pages/Content probably, this doesn't need to be top-level, i think. basically all things that affect run-time transformation of page content should go there, wdtyt?

we should remove Mermaid.tsx too

Reply:
This makes sense.

Are u thinking about removing the mermaid viewer?

Reply:
![image](https://github.com/user-attachments/assets/5b4749ce-5c0a-462d-a8aa-ca84c0a89d18)

just this empty file